go.mod

```diff
@@ -7,11 +7,11 @@ require (
 	github.com/gardener/component-spec/bindings-go v0.0.70
 	github.com/go-logr/logr v1.2.3
 	github.com/go-logr/zapr v1.2.3
-	github.com/kyma-project/module-manager v0.0.0-20221207164018-ddf69229acb6
-	github.com/kyma-project/module-manager/operator v0.0.0-20221125150159-5e84c4855249
+	github.com/kyma-project/module-manager v0.0.0-20221214222100-348f6d59ed0b
 	github.com/kyma-project/runtime-watcher/listener v0.0.0-20221129135108-60e6093eba9d
 	github.com/onsi/ginkgo/v2 v2.5.0
 	github.com/onsi/gomega v1.24.1
+	github.com/slok/go-helm-template v0.3.0
```

Comment:
this still includes the helm libraries, did we not agree to get rid of the entire chart?

Reply:
Yes. But we will do it in a separate ticket: #346